.eslintrc.js

```diff
@@ -51,64 +50,37 @@ module.exports = {
         ],
       },
     ],
-    'nonblock-statement-body-position': 'error',
-    // Airbnb restricts isNaN and isFinite which are necessary for IE 11
-    // we have to be disciplined about the usage and ensure the Number type for its
-    // arguments
-    'no-restricted-globals': ['error'].concat(confusingBrowserGlobals),
+    'no-constant-condition': 'error',
+    'no-prototype-builtins': 'off', // Use the proptype inheritance chain
```

Comment:
Why was this rule turned off?

Reply:
Do you agree with the rule? https://eslint.org/docs/rules/no-prototype-builtins. If I don't disable it, it fails in 5 places, for instance in the Modal

<img width="504" alt="Capture d’écran 2020-07-15 à 16 22 09" src="https://user-images.githubusercontent.com/3165635/87556806-63914100-c6b7-11ea-8457-8a6f5b0987fa.png">

Reply:
I'd rather disable these on a per line basis. I do follow the rationale but this rule also documents very well when it should be turned of. We can't guarantee it globally though.

E.g. `setTimeout.hasOwnProperty` is safe because `setTimeout` is a JS global that does inherit. `someObject.hasOwnProperty` is unsafe if you don't know where the object is coming from. I'd say the rule flagging `intent.hasOwnProperty` is actually a good thing to guard against because it's executed on the server.